src/main/java/com/google/sps/data/Event.java

```diff
@@ -72,15 +72,15 @@
   public Event(Entity entity) {
     this.id = (long) entity.getKey().getId();
     this.name = (String) entity.getProperty("eventName");
-    this.ownerOrgId = (long) entity.getProperty("eventOwnerOrgId");
+    //this.ownerOrgId = (long) entity.getProperty("eventOwnerOrgId");
     this.partnerNames = (ArrayList) entity.getProperty("eventPartnerNames");
     this.details = (String) entity.getProperty("eventDetails");
     this.dateAndHours = (HashMap) entity.getProperty("eventDateAndHours");
     this.contactEmail = (String) entity.getProperty("eventContactEmail");
     this.contactPhone = (String) entity.getProperty("eventContactPhone");
     this.contactName = (String) entity.getProperty("eventContactName");
-    this.creationTimeStampMillis = (long) entity.getProperty("eventCreationTimeStampMillis");
-    this.lastEditedTimeStampMillis = (long) entity.getProperty("eventLastEditTimeStampMillis");
+    this.creationTimeStampMillis = (long) entity.getProperty("creationTimeStampMillis");
```

Comment:
i think these two time fields should be removed since they shouldn't be sent to the front end